README.rst

```diff
@@ -94,7 +94,18 @@ Now open an attached file or edit an entry:
 
 |edit|
 
-Or export them to bibtex:
+If you change your mind after typing a command, there are shortcuts to
+browse (`Ctrl-b`), edit (`Ctrl-e`), edit notes (`Ctrl-q`) instead !
+
+|picker|
```

Comment:
```suggestion
The Papis picker (set using the ``picktool`` configuration option) has
helpful shortcuts to call other functionality as well (type ``F1`` for 
a complete list). It can be used to open (``Ctrl-o``), browse (``Ctrl-b``)
or edit (``Ctrl-e``) selected documents (marked with ``Ctrl-t``).
```
What do you think about this?